### ISSUE TITLE
Overly verbose output

### ISSUE THREAD

**Issue by Ryuno-Ki** [tweetback/tweetback#67]
Running the import npm run command creates lots of output.
I suspect this as the root cause:

https://github.com/tweetback/tweetback/blob/7dfcdbfe261475760d53f30f9975102ae79ab3e3/database/tweet-to-db.js#L84-L88

Since the total number of tweets found in the archive is known upfront I was wondering whether a progress bar wouldn't make more sense?

Either way it is too verbose for a screenreader, so you might want to add a `--quiet` flag to reduce the lines sent to STDOUT.

**Comment by cooljeanius** [comment]
#43 is also about that same text

**Comment by Ryuno-Ki** [comment]
True. I must have missed it as I was looking through an accessibility lense. I'm undecided whether I want to close this here or keep it because it has a slightly different perspective.

**Comment by cooljeanius** [comment]
> I'm undecided whether I want to close this here or keep it because it has a slightly different perspective.

I guess that's up to @zachleat ...
